lib/src/fixtures/spec-only-booted.yaml

```diff
@@ -6,6 +6,7 @@ spec:
   image:
     image: quay.io/centos-bootc/centos-bootc:stream9
     transport: registry
+    architecture: amd64
```

Comment:
I'm not sure I understand this one 100%.. In the other test fixture files you didn't add a spec.image.architecture. Is this one here on purpose?

Also is it intentional that this architecture is `amd64` and the one later in this same file is `arm64`?

Reply:
Nope it's a mistake, we need to add `serde(deny_unknown_fields)` here